gamedev-iitk/tower-defense
# Restyled dotnet-format *FIXED*

**Description:** (Enter a little description of the problem that this PR solves and how)
Got the restyler working. The restyler used is dotnet-format. This restylers formats the tower-defense.sln SOLUTION. 
**Issue:** Fixes #15 

**Steps to test:**
1. How should your PR be tested
1. You should omit trivial steps like "pull this branch"
1. Make a branch of master. Add a comment or a simple line in the code, which does not effects its function.
2. Make a PR from branch to master.
3. All checks are passed
